src/test/java/duke/model/user/UserTest.java

```diff
@@ -0,0 +1,23 @@
+package duke.model.user;
+
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class UserTest {
+    private User user = new User("Foo Chi Hen", 22, 100, Gender.MALE, 0, true);
```

Comment:
you may want to exclude personal data from public repositories. the choice, however, is completely yours.